dataactbroker/handlers/jobHandler.py

```diff
@@ -24,6 +24,10 @@ def getSubmissionById(self,submissionId):
         result = self.runUniqueQuery(query,"No submission with that ID","Multiple submissions with that ID")
         return result
 
+    def getSubmissionsByUserAgency(self,user,limit=5):
```

Comment:
I honestly don't know the best practice here, so just throwing out this question...if the front-end is currently wireframed to show 5 submissions, is it better to send back exactly 5 and let the web app make additional calls if/when we see a pagination story, or is it better to send back more than 5 and let the web app handle pruning down the list to 5?